cirq-core/cirq/transformers/stratify.py

```diff
@@ -48,8 +64,12 @@ def stratified_circuit(
     with operations from the same category. There is no guarantee that the
     resulting circuit will be optimally packed under this constraint.
 
+    Tagged Operations marked with any of `context.tags_to_ignore` will be treated as a separate
+    category will be left in their original moments without stratification.
+
     Args:
         circuit: The circuit whose operations should be re-arranged.
+        context: `cirq.TransformerContext` storing common configurable options for transformers.
         categories: A list of classifiers picking out certain operations.
```

Comment:
We already describe all the valid types in the description of this arg? 

```
There are several ways to specify a classifier. You can pass in a gate instance 
(e.g. `cirq.X`), a gate type (e.g. `cirq.XPowGate`), an operation instance 
(e.g. `cirq.X(cirq.LineQubit(0))`), an operation type (e.g.`cirq.CircuitOperation`), 
or an arbitrary operation predicate (e.g. `lambda op: len(op.qubits) == 2`).
```

I think it's useful to allow these types instead of directly requiring lambda's because an iterable of lambda's would be messy to write. For example:

```python
>>> categories = [cirq.X, cirq.Y, cirq.Z] # Way-1: Much cleaner 
>>> categories = [lambda op: isinstance(op.gate, cirq.X), lambda op: isinstance(op.gate, cirq.Y), lambda op: isinstance(op.gate, cirq.Z)] # Way-2: Much worse 
```